content/embeds/shard-placement-intro.md

```diff
@@ -0,0 +1,17 @@
+In Redis Enterprise Software (RS), the location of master and slave shards on the cluster nodes can impact the database and node performance.
+Master shards are always hosted separately from their slave shards.
+By default, all master shards for a database are hosted on the same node and all slave shards are hosted on a different node,
+as long as the nodes have enough resources to host them.
+
+This shard placement scheme attempts to maintain fast data retrieval because all of the data in the database is hosted together.
```

Comment:
This is the case for RoR with dense (where we prefer as many master shards on the same node), but also for RoF with sparse (where the disk becomes a bottleneck).

Reply:
I rewrote it based on your comments.